yt_dlp/options.py

```diff
@@ -42,28 +44,26 @@ def parseOpts(overrideArguments=None, ignore_config_files='if_override'):
     if ignore_config_files == 'if_override':
         ignore_config_files = overrideArguments is not None
 
+    def _load_from_config_dirs(config_dirs, package_name):
+        for config_dir in config_dirs:
+            conf_file_path = os.path.join(config_dir, package_name, 'config')
+            conf = Config.read_file(conf_file_path, default=None)
+            if conf is None:
+                conf_file_path += '.txt'
+                conf = Config.read_file(conf_file_path, default=None)
+            if conf is not None:
+                return conf_file_path, conf
+        return None, None
+
     def _readUserConf(package_name, default=[]):
```

Comment:
these are not public functions. Feel free to refactor as necessary

Reply:
also TODO: need to test the config locations all still work